<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>poky - Poky Build Tool and Metadata</title>
<meta name='generator' content='cgit v0.10.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.yoctoproject.org/cgit/cgit.cgi/poky/atom/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch?h=master' type='application/atom+xml'/>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
<script type="text/javascript">
$(document).ready(function() {
fragment = '';
if (window.location.href.lastIndexOf('?') > -1) {
	fragment = window.location.href.substring(window.location.href.lastIndexOf('?'));
}
$('.logo').append('<a href="http://git.yoctoproject.org/clean/cgit.cgi/' + window.location.pathname.substring(window.location.pathname.lastIndexOf("cgit.cgi")+9) + fragment + '">light theme</a>');
});
</script>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/cgit.cgi/'><img src='/cgit/yocto-project-transp.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/cgit.cgi/'>index</a> : <a title='poky' href='/cgit/cgit.cgi/poky/'>poky</a></td><td class='form'><form method='get' action=''>
<input type='hidden' name='id' value='1a97243bf86468a9dfd44f60c9f7563aff54b284'/><select name='h' onchange='this.form.submit();'>
<option value='1.1_M1'>1.1_M1</option>
<option value='1.1_M2'>1.1_M2</option>
<option value='1.1_M3'>1.1_M3</option>
<option value='1.1_M4'>1.1_M4</option>
<option value='1.2_M1'>1.2_M1</option>
<option value='1.2_M2'>1.2_M2</option>
<option value='1.2_M3'>1.2_M3</option>
<option value='1.2_M4'>1.2_M4</option>
<option value='1.3_M1'>1.3_M1</option>
<option value='1.3_M2'>1.3_M2</option>
<option value='1.3_M3'>1.3_M3</option>
<option value='1.3_M4'>1.3_M4</option>
<option value='1.3_M5'>1.3_M5</option>
<option value='1.3_beta'>1.3_beta</option>
<option value='1.4_M1'>1.4_M1</option>
<option value='1.4_M3'>1.4_M3</option>
<option value='1.4_M4'>1.4_M4</option>
<option value='1.4_M5'>1.4_M5</option>
<option value='1.6_M5'>1.6_M5</option>
<option value='bernard'>bernard</option>
<option value='blinky'>blinky</option>
<option value='clyde'>clyde</option>
<option value='daisy'>daisy</option>
<option value='danny'>danny</option>
<option value='danny-next'>danny-next</option>
<option value='denzil'>denzil</option>
<option value='dizzy'>dizzy</option>
<option value='dora'>dora</option>
<option value='dora-toaster'>dora-toaster</option>
<option value='dylan'>dylan</option>
<option value='edison'>edison</option>
<option value='elroy'>elroy</option>
<option value='fido'>fido</option>
<option value='green'>green</option>
<option value='jethro'>jethro</option>
<option value='krogoth'>krogoth</option>
<option value='krogoth-next'>krogoth-next</option>
<option value='laverne'>laverne</option>
<option value='master' selected='selected'>master</option>
<option value='master-next'>master-next</option>
<option value='master-next2'>master-next2</option>
<option value='morty'>morty</option>
<option value='morty-next'>morty-next</option>
<option value='pinky'>pinky</option>
<option value='purple'>purple</option>
<option value='pyro'>pyro</option>
<option value='pyro-next'>pyro-next</option>
<option value='rocko'>rocko</option>
<option value='rocko-next'>rocko-next</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>Poky Build Tool and Metadata</td><td class='sub right'>git repository hosting</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/cgit.cgi/poky/'>summary</a><a href='/cgit/cgit.cgi/poky/refs/?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>refs</a><a href='/cgit/cgit.cgi/poky/log/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch'>log</a><a class='active' href='/cgit/cgit.cgi/poky/tree/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>tree</a><a href='/cgit/cgit.cgi/poky/commit/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>commit</a><a href='/cgit/cgit.cgi/poky/diff/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>diff</a><a href='/cgit/cgit.cgi/poky/stats/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/cgit.cgi/poky/log/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch'>
<input type='hidden' name='id' value='1a97243bf86468a9dfd44f60c9f7563aff54b284'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/cgit.cgi/poky/tree/?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>root</a>/<a href='/cgit/cgit.cgi/poky/tree/meta?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>meta</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-kernel?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>recipes-kernel</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-kernel/lttng?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>lttng</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-kernel/lttng/lttng-modules?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>lttng-modules</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch</a></div><div class='content'>blob: e41124227201a9d7bb53cd6a7a5ef6aa733328e2 (<a href='/cgit/cgit.cgi/poky/plain/meta/recipes-kernel/lttng/lttng-modules/Makefile-Do-not-fail-if-CONFIG_TRACEPOINTS-is-not-en.patch?id=1a97243bf86468a9dfd44f60c9f7563aff54b284'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
<a id='n44' href='#n44'>44</a>
<a id='n45' href='#n45'>45</a>
<a id='n46' href='#n46'>46</a>
<a id='n47' href='#n47'>47</a>
<a id='n48' href='#n48'>48</a>
<a id='n49' href='#n49'>49</a>
</pre></td>
<td class='lines'><pre><code>From 1b0e574d680101105a6c1e8931c78824f5a97a42 Mon Sep 17 00:00:00 2001
From: Otavio Salvador &lt;otavio@ossystems.com.br&gt;
Date: Mon, 5 Sep 2016 17:08:56 +0000
Subject: [PATCH] Makefile: Do not fail if CONFIG_TRACEPOINTS is not enabled
Organization: O.S. Systems Software LTDA.

The lttng-modules are being pulled by the tools-profile image feature,
however, not every kernel has the CONFIG_TRACEPOINTS feature enabled.

This change makes the build do not fail when CONFIG_TRACEPOINTS is not
available, allowing it to be kept being pulled by default.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Otavio Salvador &lt;otavio@ossystems.com.br&gt;
---
 Makefile | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 8602649..75550cc 100644
--- a/Makefile
+++ b/Makefile
@@ -8,10 +8,7 @@ ifneq ($(KERNELRELEASE),)
   # and defines the modules to be built.
 
   ifdef CONFIG_LOCALVERSION	# Check if dot-config is included.
-    ifeq ($(CONFIG_TRACEPOINTS),)
-      $(error The option CONFIG_TRACEPOINTS needs to be enabled in your kernel configuration)
-    endif # CONFIG_TRACEPOINTS
-  endif # ifdef CONFIG_LOCALVERSION
+    ifneq ($(CONFIG_TRACEPOINTS),)
 
   TOP_LTTNG_MODULES_DIR := $(shell dirname $(lastword $(MAKEFILE_LIST)))
 
@@ -94,6 +91,10 @@ ifneq ($(KERNELRELEASE),)
   obj-$(CONFIG_LTTNG) += lib/
   obj-$(CONFIG_LTTNG) += tests/
 
+    else
+      $(warning The option CONFIG_TRACEPOINTS needs to be enabled in your kernel configuration)
+    endif # CONFIG_TRACEPOINTS
+  endif # ifdef CONFIG_LOCALVERSION
 else # KERNELRELEASE
 
 # This part of the Makefile is used when the 'make' command is runned in the
-- 
2.1.4

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.2 at 2018-02-19 12:16:57 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
